joebvp/cfg.py

```diff
@@ -17,13 +17,17 @@
 field=''
 homedir=os.path.expanduser('~')
 spectrum = []
-lsf='COS_LP1'
-instr=['COS','COS','COS','COS','STIS']
-gratings=['G130M','G160M','G185M','G225M','E230M']
-slits=['NA','NA','NA','NA','0.2x0.2']
-lsfranges=np.array([[1100,1400],[1400,1700],[1700,2100],[2100,2278],[2278,4100]])
-lps=['1','1','1','1','1']
-cen_wave=['1327','1600','1953','2250','2707']
+
+
+lsf='COS_LP3'
+instr=['COS','COS']
+gratings=['G130M','G160M']
+
+slits=['NA','NA']
+lsfranges=np.array([[1100,1400],[1300,1800]])
+lps=['3','3']
+cen_wave=['1327','1600']
+
 echarge=4.803204505713468e-10
```

Comment:
Sure, no problem, I came up with a patch for now but I agree it would be nice to come up with a better long-term solution.